SwiftWisdom/Rx/Rx+String.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Rx+String.swift
+//  SwiftWisdom
+//
+//  Created by Maya Saxena on 6/12/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+extension ObservableType where E == String {
+    public func ip_limitLength(to limit: Int) -> Observable<E> {
+        return self.map { $0.ip_trimmed(toLength: limit) }
+    }
+}
+
+extension ObservableType where E == String? {
+    public func ip_limitLength(to limit: Int) -> Observable<E> {
+        return self.map { $0?.ip_trimmed(toLength: limit) }
+    }
+}
+
+extension Reactive where Base: UITextField {
+    /** Limit the length of input to a text field
+
+    Usage Example:
+
+     textField.rx.ip_limitLength(to: 5)
+    */
+    public func ip_limitLength(to limit: Int) -> Disposable {
+        return text.ip_limitLength(to: limit).bind(to: text)
+    }
```

Comment:
This works great when responding to user inputs, but I noticed it doesn't work when you modify a text field's `text` property directly, like this:

```
textField.rx.ip_limitLength(to: 10).disposed(by: bag)
textField.text = "a very long string"
```

This feels like something we should account for. At the very least I think it's a good idea to add a comment to document this edge case.